README.md

```diff
@@ -1,3 +1,24 @@
+<!-- START doctoc generated TOC please keep comment here to allow auto update -->
+
+<!-- DON'T EDIT THIS SECTION, INSTEAD RE-RUN doctoc TO UPDATE -->
+
+* [rtl-css-js](#rtl-css-js)
```

Comment:
this had to be added by some script

Reply:
Yup. Could you move this down to below "This solution?" Then it will be regenerated to only include stuff below it. Also add a `## Table of Contents` above it as well.